macaw-riscv/src/Data/Macaw/RISCV/Identify.hs

```diff
@@ -62,7 +62,12 @@ matchReturn :: RISCVConstraints rv
             -> Maybe (Some (MA.AbsValue w))
```

Comment:
Do we get to fix the FIXME comment on this function with this change?

Reply:
Yep!  Good catch.  I removed the FIXME.